tensorboard/webapp/core/BUILD

```diff
@@ -7,15 +7,20 @@ load("//tensorboard/defs:defs.bzl", "tf_ts_library")
 ng_module(
     name = "core",
     srcs = [
-        "core.actions.ts",
-        "core.effects.ts",
+        "actions/core.actions.ts",
+        "actions/index.ts",
+        "effects/core.effects.ts",
+        "effects/index.ts",
         "core.module.ts",
-        "core.reducers.ts",
-        "core.service.ts",
+        "store/core.reducers.ts",
```

Comment:
Consider having BUILD file per directory so they can be depended on without the UI parts. Sooner or later, we will, for testing (will be limited to `testing.ts` mostly), have dependency dag pointing across folder.